docs/001-devices.apib

```diff
@@ -284,17 +284,17 @@ See information about an IP address assignment.
 
 + Model
 
-        {
-          "id": "3c29133a-d10c-4b86-86d4-69b9b16eadb5",
-          "address": "75.72.66.16", // IP address for IPv4 || null for IPv6
-          "network": "75.72.66.16", // network address
-          "address_family": 4, // 4 || 6 depending of the address family
-          "netmask": "255.255.255.255",
-          "public": true, // true for public IP or false for private IP
-          "cidr": 32,
-          "assigned_to": { "id":"8738dd01-7246-443a-95b8-c8c493bd2c9b" },
-          "href": "/ips/3c29133a-d10c-4b86-86d4-69b9b16eadb5"
-        }
+            {
+              "id": "3c29133a-d10c-4b86-86d4-69b9b16eadb5",
+              "address": "75.72.66.16", // IP address for IPv4 || null for IPv6
```

Comment:
I'm not sure that we need these comments here--the response should make sense to someone who is familiar with IP addressing already, I think.